content/user/apps/Route-coverage.md

```diff
@@ -0,0 +1,40 @@
+<!--
+title: "Route Coverage"
+description: "Overview of setting up environments"
+tags: "user UI applications route coverage exercised vulnerabilities"
+-->
+
+## About Route Coverage
+
+An application “route” is a combination of three distinct data points: the normalized URL of the route, the HTTP verb associated with the request (e.g., Get or Post) and the method signature of the controller action. When you consistently exercising each route in your application, the Contrast agent can successfully Assess and Protect the surface layer of your application, and discover vulnerabilities. 
```

Comment:
The route URLs may not be normalized. So we should change "normalized URL" -> "URL".